src/renderer/components/ft-list-video/ft-list-video.scss

```diff
@@ -8,3 +8,22 @@
   // Makes img element sized correctly before image loading starts
   aspect-ratio: 16/9 auto;
 }
+
+.videoTagLine {
+  margin-block-start: 3px;
+}
+
+.videoTag {
+  padding: 3px;
+  margin-inline: 2px;
+  background-color: var(--secondary-card-bg-color);
+  font-size: 10px;
+  border-radius: 2px;
+  display: inline-block;
+  font-weight: 500;
+  white-space-collapse: collapse;
```

Comment:
Again not request to change but suggestion for future changes
IMO the rules are easier to read when grouped (and sorted is even better) like (comment is optional)

```css
.videoTag {
  /* Spacing/outward stuff */
  margin-inline: 2px;
  padding: 3px;

  /* Self */
  display: inline-block;
  background-color: var(--secondary-card-bg-color); /* Normally I group this with color but it's absent here */

  /* Content like text */
  font-size: 10px;
  font-weight: 500;
  white-space-collapse: collapse;
```